File: android/app/src/main/java/com/drive/app/model/EventsNearbyFilter.java
package com.drive.app.model;

/**
 * Created by Szymon Gajewski on 18.01.2016.
 */
public class EventsNearbyFilter {

    private int maxDaysLeft;
    private int maxDistance;
    private double latitude;
    private double longitude;

    public EventsNearbyFilter(int maxDaysLeft, int maxDistance, double latitude, double longitude) {
        this.maxDaysLeft = maxDaysLeft;
        this.maxDistance = maxDistance;
        this.latitude = latitude;
        this.longitude = longitude;
    }

    public int getMaxDaysLeft() {
        return maxDaysLeft;
    }

    public void setMaxDaysLeft(int maxDaysLeft) {
        this.maxDaysLeft = maxDaysLeft;
    }

    public int getMaxDistance() {
        return maxDistance;
    }

    public void setMaxDistance(int maxDistance) {
        this.maxDistance = maxDistance;
    }

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }
}

File: api/DriveApi/Model/EventsNearbyFilter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace DriveApi.Model
{
    [Serializable]
    public class EventsNearbyFilter : GeoPoint
    {
        private int maxDaysLeft;
        private int maxDistance;

        public EventsNearbyFilter()
        {
            //
        }

        public EventsNearbyFilter(double latitude, double longitude, int maxDaysLeft, int maxDistance)
        {
            this.MaxDaysLeft = maxDaysLeft;
            this.MaxDistance = maxDistance;
            this.Latitude = latitude;
            this.Longitude = longitude;
        }

        public int MaxDaysLeft
        {
            get { return maxDaysLeft; }
            set { maxDaysLeft = value; }
        }

        public int MaxDistance
        {
            get { return maxDistance; }
            set { maxDistance = value; }
        }
    }
}